yarn.lock

```diff
@@ -1406,9 +1410,9 @@ cors@^2.8.4:
     object-assign "^4"
     vary "^1"
 
-"credstash@git+https://github.com/KensoDev/node-credstash.git#hotfix/credstash":
+"credstash@https://github.com/KensoDev/node-credstash.git#hotfix/credstash":
```

Comment:
is there some hidden meaning behind this change? (and any idea why it's a hotfix?)

Reply:
I don't think so? yarn finds the latest version of a package, seems like some new path for this.

Reply:
We should no longer have credstash as a dependency at all.